src/main/index.ts

```diff
@@ -59,6 +59,33 @@ import { UserStore } from "../common/user-store";
 import { WeblinkStore } from "../common/weblink-store";
 import { ExtensionsStore } from "../extensions/extensions-store";
 import { FilesystemProvisionerStore } from "./extension-filesystem";
+import { CaptureConsole, Dedupe, Offline } from "@sentry/integrations";
+import * as Sentry from "@sentry/electron/dist/main";
+
+if (dsnIsValid) {
+  Sentry.init({
```

Comment:
try/catch here?

Reply:
Added https://github.com/lensapp/lens/pull/3252/commits/1bcf1b17ea92191b3c48046ecce259fc6f98a89a to improve `dsnIsValid` check.

Reply:
So the problem was that the dsn wasn't a valid Sentry endpoint? `Sentry.init()` won't throw for other reasons (like Sentry endpoint is not reachable)? (Is this even where that exception that I encountered was thrown?)

Reply:
The error is thrown because it’s not a valid Sentry address.

Good point about the not reachable address, I am not sure, need to test by turning off WiFi

Reply:
I tested, seems ok. When wifi comes back, errors are sent to sentry fine.

My concern here is if `Sentry.init()` throws an exception and it is not caught then Lens doesn't start (at least the UI doesn't show up) like what happened before the dnsIsValid fix

Reply:
Ok, I will add a `try..catch` with `console.error`.

Reply:
Refactored and add `try..catch` for `Sentry.init()` https://github.com/lensapp/lens/pull/3252/commits/2361f219dafbed66fef538805daf9434238ba5b7